k8s/prod/common/divorce/div-cos.yaml

```diff
@@ -31,10 +31,14 @@ spec:
       environment:
         MANAGEMENT_ENDPOINT_HEALTH_CACHE_TIMETOLIVE: "4000"
         IDAM_API_URL: "https://idam-api.platform.hmcts.net"
-        FEATURE_DN_REFUSAL: "true"
+        FEATURE_DN_REFUSAL: true
         FEATURE_PAPER_UPDATE: true
         SOLICITOR_DN_REJECT_AND_AMEND: true
         PAY_BY_ACCOUNT: true
+        FEATURE_RESP_SOLICITOR_DETAILS: true
```

Comment:
@yarpo Just a reminder that the LD toggle has to be turned on also for prod for this feature